common-compose.yml

```diff
@@ -1,35 +1,39 @@
 # MySQL designed for container-native deployment on Joyent's Triton platform.
```

Comment:
Ideally we'd eliminate the `common-compose.yml` entirely. We dropped it in our more recent blueprints because eliminating the use of `--link` for Consul meant that the main `docker-compose.yml` could be the common parent. https://github.com/autopilotpattern/couchbase is a good example.